chart/values.yaml

```
@@ -526,8 +534,11 @@ etcd:
     storageClass: "mayastor-etcd-localpv"
     # -- Volume size
     size: 2Gi
+  persistentVolumeClaimRetentionPolicy:
     # -- PVC's reclaimPolicy
-    reclaimPolicy: "Delete"
+    enabled: true
```

Let's leave as false since Retain is default anyway?

```suggestion
    enabled: false
```